jdl/validators/enum-validator.spec.ts

```diff
@@ -56,7 +56,7 @@ describe('jdl - EnumValidator', () => {
       context('with a reserved class name as name', () => {
         it('should fail', () => {
           expect(() => {
-            validator.validate(new JDLEnum({ name: 'Catch' }));
+            validator.validateBusiness(new JDLEnum({ name: 'Catch' }));
```

Comment:
Why did this method name need to change?

Reply:
I've split basic jdl validation and business validation.

Currently if blueprints config is passed, all validation is skipped:
https://github.com/jhipster/generator-jhipster/blob/da7bc6f17a99f206112a89886af43cf5db1c6fcd/jdl/validators/jdl-with-application-validator.ts#L49-L59

Now only business validation are ignored:
https://github.com/jhipster/generator-jhipster/blob/69e8c39793d6a02a302880cc3fda3a343af8eb4a/jdl/validators/jdl-with-application-validator.ts#L51-L65

Reply:
I will add options instead of splitting.
It’s cleaner.

Reply:
I suppose this is resolved